jujuclient/controllers.go

```diff
@@ -28,6 +28,15 @@ func ReadControllersFile(file string) (*Controllers, error) {
 		if os.IsNotExist(err) {
 			return &Controllers{}, nil
 		}
+		if os.IsPermission(err) {
+			u, userErr := utils.LocalUsername()
+			if userErr != nil {
+				return nil, err
+			}
+			if ok, fileErr := utils.IsFileOwner(file, u); fileErr == nil && !ok {
+				err = errors.Annotatef(err, "it appears the ownership of the file is not the same as the current user")
```

Comment:
nit: s/it appears the //

Can we specify which file here?  Or is it included in err?

Reply:
The file is specified in the error, so I didn't want to double it up.